components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -101,4 +101,18 @@
  */
 - (void)setContentOffsetY:(CGFloat)contentOffsetY animated:(BOOL)animated;
 
+/**
+ Animates the navigation drawer to preferred content height. This ignores the initial height of 50%
+ and animates to full screen if content is large enough. If content is smaller than the screen
```

Comment:
if content is smaller than the screen will it not animate to the preferredContentHeight height?